catalog/rest/catalog-rest-endpoint/src/test/java/org/codice/ddf/endpoints/rest/TestRestEndpoint.java

```diff
@@ -909,7 +909,7 @@ protected String mockTestSetup(CatalogFramework framework, TestType testType)
       case RESOURCE_TEST:
         transformer = "resource";
         /* FALLTHRU */
-
+        // fall through
```

Comment:
❓ Did you just add a second fall through comment because the one on line 911 isn't understood by the static analysis?

Reply:
Yes. ErrorProne has an expectation of how the comment should look. I contemplated removing the first comment, but suspect it might be silencing one of our _other_ static analysis tools.